Modules/Filtering/BiasCorrection/test/itkN4BiasFieldCorrectionImageFilterTest.cxx

```diff
@@ -164,15 +164,44 @@ N4(int argc, char * argv[])
     maskImage->DisconnectPipeline();
   }
 
-  // instantiate N4 and assign variables not exposed to the user in this test.
+  // Instantiate N4 and assign variables not exposed to the user in this test.
   using CorrecterType = itk::N4BiasFieldCorrectionImageFilter<ImageType, MaskImageType, ImageType>;
   typename CorrecterType::Pointer correcter = CorrecterType::New();
-  correcter->SetSplineOrder(3);
+
+  EXERCISE_BASIC_OBJECT_METHODS(correcter, N4BiasFieldCorrectionImageFilter, ImageToImageFilter);
```

Comment:
Did you forget the `ITK_` prefix, or did you leave it out on purpose? That might be a reason for the failure 😄

Reply:
Thanks @dzenanz 🦅 . Fixed in 155b729.